CRITICAL :   root :   13-06-2025 07:16:39 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
   ERROR :   root :   13-06-2025 07:20:04 PM :   trigger  :File does not exist. Please provide a valid file. 
   ERROR :   root :   13-06-2025 07:21:35 PM :       opp  :File transfer failed with error: Suppressing FBS.
Connecting...\done
Sending "/home/engineer/Pictures/a.png"... |/-\|/-\|/-\|/-\|/-\|/-\|/-\done
Disconnecting...|done 
CRITICAL :   root :   13-06-2025 10:58:46 PM :       gap  :org.bluez.Error.Failed: No discovery started 
CRITICAL :   root :   13-06-2025 11:01:02 PM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   13-06-2025 11:01:17 PM :       gap  :org.bluez.Error.InProgress: In Progress 
CRITICAL :   root :   13-06-2025 11:02:03 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
CRITICAL :   root :   13-06-2025 11:02:14 PM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
CRITICAL :   root :   13-06-2025 11:03:45 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
CRITICAL :   root :   13-06-2025 11:04:09 PM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
CRITICAL :   root :   13-06-2025 11:08:20 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
CRITICAL :   root :   13-06-2025 11:08:45 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
CRITICAL :   root :   13-06-2025 11:09:42 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
CRITICAL :   root :   13-06-2025 11:10:31 PM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
   ERROR :   root :   13-06-2025 11:21:16 PM :       opp  :File transfer failed with error: Suppressing FBS.
Connecting...\done
Sending "/home/engineer/a.py"... |/done
Disconnecting...-done 
   ERROR :   root :   13-06-2025 11:47:40 PM :       opp  :Could not determine OBEX OPP channel from device. 
   ERROR :   root :   13-06-2025 11:48:15 PM :       opp  :Could not determine OBEX OPP channel from device. 
   ERROR :   root :   13-06-2025 11:51:24 PM :       opp  :Could not determine OBEX OPP channel from device. 
CRITICAL :   root :   14-06-2025 12:05:14 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:09:17 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:09:57 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:10:43 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:12:55 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:18:59 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:21:00 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
   ERROR :   root :   14-06-2025 01:07:21 AM :       opp  :Could not determine OBEX OPP channel from device. 
   ERROR :   root :   14-06-2025 01:12:57 AM :   trigger  :File does not exist. Please provide a valid file. 
   ERROR :   root :   14-06-2025 01:33:22 AM :      pbap  :Failed to pull contact file: Scanning for  ...
Found: B4:31:61:D4:43:5E
Connecting...failed: connect
The user may have rejected the transfer: Connection refused
 
CRITICAL :   root :   14-06-2025 11:32:06 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 11:34:44 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 11:34:51 AM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 11:34:57 AM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 11:35:02 AM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 11:35:02 AM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 11:35:03 AM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 11:35:03 AM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 11:36:44 AM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
CRITICAL :   root :   14-06-2025 12:30:33 PM :     utils  :name 'bus' is not defined 
CRITICAL :   root :   14-06-2025 01:00:35 PM :       gap  :org.bluez.Error.InProgress: Operation already in progress 
CRITICAL :   root :   14-06-2025 01:02:51 PM :       gap  :org.bluez.Error.Failed: No discovery started 
   ERROR :   root :   14-06-2025 01:25:10 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:28:22 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:31:22 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:31:26 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:31:29 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:31:39 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:31:41 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 01:31:43 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
CRITICAL :   root :   14-06-2025 01:38:57 PM :       gap  :org.bluez.Error.NotReady: Resource Not Ready 
   ERROR :   root :   14-06-2025 08:13:49 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 08:16:07 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 08:17:00 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   14-06-2025 08:17:09 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
CRITICAL :   root :   14-06-2025 08:18:45 PM :       gap  :org.bluez.Error.Failed: No discovery started 
   ERROR :   root :   14-06-2025 08:22:52 PM :      pbap  :Failed to pull contact file: Scanning for  ...
Found: 5C:F3:70:79:93:F8
Connecting...\failed: send UUID
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
CRITICAL :   root :   16-06-2025 11:42:13 AM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 11:59:23 AM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
CRITICAL :   root :   16-06-2025 12:01:56 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
   ERROR :   root :   16-06-2025 12:23:58 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:37:29 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:37:35 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:37:41 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:38:04 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:41:21 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:51:39 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:53:21 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 12:59:46 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:00:09 PM :      a2dp  :Failed to go to previous track.
Command '['playerctl', 'previous']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:00:20 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:00:27 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:00:30 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:00:30 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:02:07 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 01:03:16 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   16-06-2025 02:47:17 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 02:47:24 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
   ERROR :   root :   16-06-2025 02:47:59 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 02:48:47 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 02:49:43 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 02:49:56 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 02:55:13 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for  ...
Found: F8:9E:94:33:AF:17
Connecting...failed: connect
The user may have rejected the transfer: Connection refused
 
   ERROR :   root :   16-06-2025 02:55:52 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for yes ...
Seen: F8:9E:94:33:AF:17
Seen: CC:15:31:05:BD:91
Seen: A4:42:3B:22:81:D3
Seen: 00:D4:9E:A8:1E:F6
Seen: A4:42:3B:22:ED:26
Seen: A0:80:69:EB:41:D1
Seen: 28:DF:EB:43:DE:0A
Seen: F8:9E:94:33:BD:04
Seen: B4:31:61:2A:C3:8E
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   16-06-2025 02:56:18 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for yes ...
Seen: A4:42:3B:22:81:D3
Seen: F8:9E:94:33:AF:17
Seen: B4:31:61:2A:C3:8E
Seen: F8:9E:94:33:BD:04
Seen: CC:15:31:05:BD:91
Seen: A4:42:3B:22:ED:26
Seen: A0:80:69:EB:41:D1
Seen: 28:DF:EB:43:DE:0A
Seen: 7C:21:4A:43:4C:83
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
CRITICAL :   root :   16-06-2025 02:59:31 PM :       gap  :org.bluez.Error.DoesNotExist: Does Not Exist 
CRITICAL :   root :   16-06-2025 03:01:43 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:01:51 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:01:56 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:03:01 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:03:08 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:03:12 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:04:50 PM :       gap  :org.bluez.Error.DoesNotExist: Does Not Exist 
CRITICAL :   root :   16-06-2025 03:10:40 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:10:48 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:10:54 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
CRITICAL :   root :   16-06-2025 03:11:23 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
   ERROR :   root :   16-06-2025 03:24:50 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for  ...
Found: 48:74:12:31:17:B0
Connecting...failed: connect
The user may have rejected the transfer: Connection refused
 
   ERROR :   root :   16-06-2025 03:28:37 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for  ...
Found: 5C:F3:70:79:93:F8
Connecting...\failed: send UUID
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   16-06-2025 03:38:49 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for  ...
Found: A4:42:3B:22:ED:26
Connecting...failed: connect
The user may have rejected the transfer: Connection refused
 
   ERROR :   root :   16-06-2025 03:40:31 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for  ...
Found: F8:9E:94:33:AF:17
Connecting...failed: connect
The user may have rejected the transfer: Connection refused
 
   ERROR :   root :   16-06-2025 04:26:14 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   16-06-2025 05:16:33 PM :       gap  :org.bluez.Error.AlreadyExists: Already Exists 
   ERROR :   root :   16-06-2025 06:04:35 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 06:04:54 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 06:04:55 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 06:05:02 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 06:05:03 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 06:05:05 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 06:05:16 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   16-06-2025 06:09:52 PM :       gap  :org.bluez.Error.ConnectionAttemptFailed: Page Timeout 
CRITICAL :   root :   16-06-2025 06:10:28 PM :       gap  :org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken. 
   ERROR :   root :   16-06-2025 06:53:29 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   16-06-2025 07:02:37 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   16-06-2025 07:03:28 PM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
   ERROR :   root :   17-06-2025 11:43:13 AM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 11:44:06 AM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 11:44:55 AM :      a2dp  :Failed to go to previous track.
Command '['playerctl', 'previous']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 11:45:09 AM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:04:50 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:05:00 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:05:03 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:05:04 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:05:04 PM :      a2dp  :Failed to go to previous track.
Command '['playerctl', 'previous']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:05:05 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   17-06-2025 05:06:19 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   17-06-2025 07:01:29 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
CRITICAL :   root :   17-06-2025 07:24:01 PM :       gap  :org.freedesktop.DBus.Error.UnknownObject: Method "Connect" with signature "" on interface "org.bluez.Device1" doesn't exist
 
   ERROR :   root :   18-06-2025 12:46:24 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 12:46:55 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL : Classic_Bluetooth_Profile.hfp :   18-06-2025 12:47:21 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
CRITICAL :   root :   18-06-2025 12:47:30 PM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
CRITICAL :   root :   18-06-2025 12:48:16 PM :       gap  :org.bluez.Error.Failed: Resource temporarily unavailable 
   ERROR :   root :   18-06-2025 01:00:04 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:12:42 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:15:20 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:22:37 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   18-06-2025 01:28:28 PM :       gap  :org.bluez.Error.Failed: Host is down 
CRITICAL :   root :   18-06-2025 01:28:40 PM :       gap  :org.bluez.Error.Failed: Host is down 
   ERROR :   root :   18-06-2025 01:33:12 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:35:26 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:35:39 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:35:51 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:37:19 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:38:43 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:39:25 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:39:40 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:39:49 PM :      a2dp  :Failed to skip track.
Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:45:28 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:45:37 PM :      a2dp  :Failed to go to previous track.
Command '['playerctl', 'previous']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:45:53 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:46:14 PM :      a2dp  :Failed to restart playlist.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:46:26 PM :      a2dp  :Failed to play media.
Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:47:33 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:58:01 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 01:59:42 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:05:45 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:05:56 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:06:59 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:02 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:03 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:12 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:13 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:14 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:16 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:17 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:07:18 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:09:07 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:09:10 PM :      a2dp  :Failed to set volume.
Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:09:11 PM :      a2dp  :Failed to set volume.
Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:11:15 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:11:30 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:11:30 PM :      a2dp  :Failed to set volume.
Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:11:35 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:11:35 PM :      a2dp  :Failed to set volume.
Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:12:11 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:12:58 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:13:00 PM :      a2dp  :Failed to pause media.
Command '['playerctl', 'pause']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:17:45 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:17:58 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:18:21 PM :      a2dp  :Failed to get volume: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 02:27:08 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
CRITICAL :   root :   18-06-2025 02:31:47 PM :       gap  :org.bluez.Error.Failed: Input/output error 
CRITICAL : Classic_Bluetooth_Profile.hfp :   18-06-2025 02:32:50 PM :   trigger  :org.freedesktop.DBus.Error.UnknownObject: Method "Get" with signature "ss" on interface "org.freedesktop.DBus.Properties" doesn't exist
 
   ERROR :   root :   18-06-2025 02:35:11 PM :      pbap  :Failed to fetch phonebook.vcf: Unknown UUID PBAP
Scanning for  ...
Found: C8:E2:65:65:C9:70
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   18-06-2025 03:28:37 PM :      pbap  :❌ Failed to fetch pb.vcf: Unknown UUID PBAP
Scanning for  ...
Found: CC:15:31:FB:F8:D2
Connecting...\failed: send UUID
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   18-06-2025 03:40:10 PM :      pbap  :Failed to fetch pb.vcf:
Suppressing FBS.
Scanning for  ...
Found: CC:15:31:05:CC:6E
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   18-06-2025 03:44:20 PM :      pbap  :Failed to fetch pb.vcf:
Suppressing FBS.
Scanning for  ...
Found: C8:E2:65:65:C9:70
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   18-06-2025 03:47:05 PM :      pbap  :Exception in fetch_contacts: local variable 'pbap_found' referenced before assignment 
Traceback (most recent call last):
  File "/home/engineer/project_developemnt_folder/Bluetooth-Features-main/Classic_Bluetooth_Profile/pbap.py", line 42, in fetch_contacts
    if pbap_found and "Channel:" in line:
UnboundLocalError: local variable 'pbap_found' referenced before assignment
   ERROR :   root :   18-06-2025 03:49:52 PM :      pbap  :❌ Failed to fetch pb.vcf:
Suppressing FBS.
Connecting...\failed: send UUID
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
Connecting...failed: connect
unknown error on connect
Still trying to connect
 
   ERROR :   root :   18-06-2025 05:22:54 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 05:50:08 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 05:51:03 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 05:52:49 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:34:32 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:35:49 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:37:13 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:37:55 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:38:58 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:40:55 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:41:44 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:43:23 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:46:49 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:47:16 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:47:23 PM :      a2dp  :Playback error: set_volume() missing 1 required positional argument: 'level' 
   ERROR :   root :   18-06-2025 06:48:06 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 06:48:40 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:13:56 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:14:12 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:14:12 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:14:13 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:14:13 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:14:14 PM :      a2dp  :Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:13 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:15 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:18 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:19 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:20 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:21 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:22 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:27 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:27 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:28 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:29 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:50:31 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:53:00 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:53:00 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:53:02 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:53:02 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:53:04 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:53:04 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:54:06 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:54:06 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:54:07 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:54:07 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:54:08 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:54:08 PM :      a2dp  :❌ Could not get default sink: Command '['pactl', 'get-default-sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:32 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:34 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:35 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:36 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:38 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:39 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:40 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:42 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:56:42 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:59:32 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:59:33 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:59:34 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:59:48 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:59:49 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 07:59:51 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:00:04 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:00:48 PM :      a2dp  :❌ Get volume failed: Command '['pactl', 'get-sink-volume', 'bluez_sink.A4_05_6E_1F_04_67.a2dp_sink']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:34:33 PM :      a2dp  :❌ Skip failed: Command '['playerctl', 'next']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:34:50 PM :      a2dp  :❌ Previous track failed: Command '['playerctl', 'previous']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:35:14 PM :      a2dp  :❌ Play failed: Command '['playerctl', 'play']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:35:29 PM :      a2dp  :❌ Media control error: volume_up() missing 1 required positional argument: 'player' 
   ERROR :   root :   18-06-2025 08:35:40 PM :      a2dp  :❌ Media control error: volume_up() missing 1 required positional argument: 'player' 
   ERROR :   root :   18-06-2025 08:40:22 PM :      a2dp  :❌ Playback control error: volume_up() takes 0 positional arguments but 1 was given 
   ERROR :   root :   18-06-2025 08:40:48 PM :      a2dp  :❌ Playback control error: get_volume() takes 0 positional arguments but 1 was given 
   ERROR :   root :   18-06-2025 08:53:02 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:53:07 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:53:11 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:53:12 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:53:13 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:53:13 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
   ERROR :   root :   18-06-2025 08:53:15 PM :      a2dp  :❌ Get system volume failed: Command '['pactl', 'get-sink-volume', '@DEFAULT_SINK@']' returned non-zero exit status 1. 
